modules/web-mercator/src/web-mercator-viewport.js

```diff
@@ -58,7 +61,7 @@ export default class WebMercatorViewport {
       width,
       height,
       pitch,
-      altitude,
+      altitude: fovy ? 0.5 / Math.tan(0.5 * fovy * DEGREES_TO_RADIANS) : altitude,
```

Comment:
Don't you need to specify fovy and altitude independently? Why overwrite altitude here?

Reply:
The way I understand it, then altitude is just another way of thinking about the field of view, when we are talking about the projection matrix. When the view matrix is calculated below, `altitude` is used

Reply:
This can get very confusing. `getProjectionMatrix` and `getProjectionParameters` should just accept `fovy`. We should not code the altitude <-> fov logic into viewports.